microsoftgraph/microsoft-graph-explorer-v4
# Feature: input hints

## Overview

Fixes #1062 


### Demo

https://jolly-sand-0ac78c710-1104.centralus.azurestaticapps.net

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* Click any of the sidebar samples
* Notice an information icon adjacent to the query you selected.
* Click on the icon and notice a callout with more information about the query you are running
